vh_patterns_dataset.py

```diff
@@ -535,19 +535,196 @@
+                                   (0,B_H_R_NT):(B_H_R_NT,3),(0,'e'):('e',4),(0,'u'):('u',4),
+                                   (1,'?'):('?',1),(1,'i'):('i',1),(1,B_H_R_NT):(B_H_R_NT,3),
+                                   (1,'u'):('u',4),(1,B_M_R_NT):(B_M_R_NT,1),(1,'a'):(B_M_R_NT,1),
+                                   (1,'e'):(B_M_R_NT,1),(1,'o'):(B_M_R_NT,1),
+                                   (2,'?'):('?',2),(2,'i'):('i',2),(2,B_H_R_NT):(B_H_R_NT,3),
+                                   (2,'u'):('u',4),(2,B_M_R_NT):('o',2),(2,'o'):('o',2),
+                                   (2,'e'):('o',2),(2,'a'):('o',2),
+                                   (3,'?'):('?',3),(3,'i'):('i',3),(3,'a'):('a',3),
+                                   (3,'e'):('a',3),(3,'o'):('a',3),(3,B_M_R_NT):('a',3),
+                                   (3,'u'):(B_H_R_NT,3),(3,B_H_R_NT):(B_H_R_NT,3),
+                                   (4,'?'):('?',4),(4,'i'):('i',4),(4,'e'):('e',4),
+                                   (4,'o'):('e',4),(4,'a'):('e',4),(4,B_M_R_NT):('e',4),
+                                   (4,'u'):('u',4),(4,B_H_R_NT):('u',4)},
+                    'preprocess_req': False,
+                    'postprocess_req': False,
+                    'left_subseq': True,
+                    'hyphenate_suffix': False,
+                    'preprocess_dets':None,
+                    'postprocess_dets':None,
+                    'notes': ['Mongolic language','Progressive ATR/RTR and labial harmony','i is transparent for both types of harmony','/u,'+B_H_R_NT+'/ block [+labial] spreading such that following vowels are [-labial]','I did not find data of the form a...B_M_R_NT or e...o, so I assumed [-Labial] spreads progressively, just as [+Labial does]'],
+                    'harmony_feature':['ATR/RTR','Labial'],
+                    'sc':True,
+                    'dr':False,
+                    'transparent':['/i/ for ATR/RTR and Labial'],
+                    'opaque':['/u, B_H_R_NT/ are blockers for [+labial], all that follows is [-labial]']
+                },
+            19:
+                {
+                    'name': 'Dagur Mongolian harmony',
+                    'states': {0:'',1:'',2:'',3:''},
+                    'alphabet': ['i',schwa,'u','a',B_M_R_NT], 
+                    'transitions': {(0,'?'):('?',0),(0,'i'):('i',0),
+                                   (0,B_M_R_NT):(B_M_R_NT,1),(0,schwa):(schwa,2),(0,'u'):('u',2),
+                                   (0,'a'):('a',3),
+                                   (1,'?'):('?',1),(1,'i'):('i',1),(1,B_M_R_NT):(B_M_R_NT,1),
+                                   (1,schwa):(B_M_R_NT,1),(1,'a'):(B_M_R_NT,1),(1,'u'):(B_M_R_NT,1),
+                                   (2,'?'):('?',2),(2,'i'):('i',2),(2,schwa):(schwa,2),(2,'u'):('u',2),
+                                   (2,'a'):(schwa,2),(2,B_M_R_NT):('u',2),
+                                   (3,'?'):('?',3),(3,'i'):('i',3),(B_M_R_NT3,'a'):('a',3),(3,schwa):('a',3),
```

Comment:
This is probably `3`, not `B_M_R_NT3`, but if you could confirm I'm going to make that edit ony my end.